documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -55,3 +55,38 @@ spec:
       send.buffer.bytes: 131072
     # ...
 ----
+== Capacity configuration
+
+Cruise Control uses capacity limits to determine if goals related to broker resources are being violated.
+
+Capacity limits for broker resources can be specified in the `capacity` property in `Kafka.spec.cruiseControl`.
+Capacity limits can be set for the following broker resources in the described units:
+
+* `disk`  - Disk storage in MB
+* `cpu`   - CPU utilization as a percent (0-100)
+* `nwIn`  - Network inbound throughput in KB/s
+* `nwOut` - Newtork outbound throughput in KB/s
```

Comment:
You should change here to `networkIn` and `networkOut` if you accepted my suggestion to have less cryptic names before :-)